source/index.pug

```diff
@@ -37,28 +38,28 @@ html
       .col-md-6
         include:markdown markdown/what-we-do/description.md
 
-  .bg-gray.m-y-2
+  .bg-gray.m-y-3
     .container.p-y-3
       .row
         .col-sm-12
           h2.center-text Mozilla Clubs for Women and Girls
       .row.p-y-3
+        .col-sm-12.col-md-4.wide-only
+          img(src='images/clubs/clubs-inside.jpg')
+        .col-sm-12.col-md-4.wide-only
+          img(src='images/clubs/clubs-outside.jpg')
         .col-sm-12.col-md-4
-          img(src='images/FPO.svg')
-        .col-sm-12.col-md-4
-          img(src='images/FPO.svg')
-        .col-sm-12.col-md-4
-          img(src='images/FPO.svg')
+          img(src='images/clubs/clubs-standing-presenter.jpg')
       .row
         .col-sm-12.col-md-8
           div
             include:markdown markdown/clubs/summary.md
         .col-sm-12.col-md-4
-          .bg-blue.p-x-2.p-y-2.partial-cta
+          .bg-blue.partial-cta
             include:markdown markdown/clubs/cta.md
             a.btn.btn-white(href='https://learning.mozilla.org/clubs') Learn More
 
-  .container.m-y-2
+  .container.m-y-3.our-work
```

Comment:
For the class wrappers around partials I've been using the namespace convention `partial-PARTIALNAMEHERE` so that we don't have collisions later if and when we add more pages.